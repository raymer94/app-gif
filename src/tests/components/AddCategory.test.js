import { shallow } from "enzyme";
import AddCategory from "../../Components/AddCatergory";

describe("pruebas en <AddCategory/>", () =>{
    const wrapper = shallow(<AddCategory/>);

    test("debe mostrar el componente", ()=>{
        expect(wrapper).toMatchSnapshot();
    });
})