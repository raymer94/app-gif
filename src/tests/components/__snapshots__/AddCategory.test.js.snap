// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pruebas en <AddCategory/> debe mostrar el componente 1`] = `
<form
  onSubmit={[Function]}
>
  <input
    onChange={[Function]}
    type="text"
    value=""
  />
</form>
`;
